yarn-project/types/src/tx/tx_receipt.ts

```diff
@@ -78,3 +77,39 @@ export class TxReceipt {
     return new TxReceipt(txHash, status, error, blockHash, blockNumber, contractAddress);
   }
 }
+
+/**
+ * Information useful for testing purposes included in the receipt when the test flag is set to true in `WaitOpts`.
+ */
+interface TestInfo {
```

Comment:
As said earlier, something like `DebugInfo` seems more telling, this makes me think more of whether it passed a test or not 🤔

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/3221/commits/0b51791a21a775fe7675220b1f2213bc4ef07e19